lib/actions/file.js

```diff
@@ -8,7 +8,7 @@ exports.uploadFile = function(b2, args) {
     var uploadUrl = args.uploadUrl;
     var uploadAuthToken = args.uploadAuthToken;
     // Previous versions used filename (lowercase), so support that here
-    var fileName = utils.getUrlEncodedFileName(typeof args.fileName !== "undefined" ? args.fileName : args.filename);
+    var fileName = utils.getUrlEncodedFileName(typeof args.fileName !== 'undefined' ? args.fileName : args.filename);
```

Comment:
I'd say we can reduce it to:
```js
var fileName = utils.getUrlEncodedFileName(args.fileName || args.filename);
```

Reply:
It was originally that but I changed it to a `typeof` check because the `||` wouldn't work if `fileName` was `'0'`.

Reply:
agreed, if someone uses an integer we have a different problem :) I prefer the brevity

Reply:
Also, the string `'0'` would be chosen:
`'0' || true == '0'`

Reply:
Ah interesting, I always thought it was casted into `0` in that situation. The more you know! I reverted back and squashed into one commit.